src/components/atoms/SpacesDropdown/SpacesDropdown.scss

```diff
@@ -0,0 +1,63 @@
+@import "scss/constants";
+
+$dropdown-item-icon-size: 20px;
+
+.SpacesDropdown {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  &__value {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+
+    font-size: $font-size--lg;
+    font-weight: $font-weight--700;
+    line-height: 19px;
+  }
+
+  &__item {
+    display: flex;
+    align-items: center;
+    padding: $spacing--sm $spacing--md;
+
+    text-decoration: none;
+    font-weight: $font-weight--400;
+    font-size: $font-size--md;
+    line-height: 17px;
+
+    &:hover {
+      background: opaque-black(0.05);
+    }
+
+    &-icon {
+      min-width: $dropdown-item-icon-size;
+      height: $dropdown-item-icon-size;
+      margin-right: $spacing--sm;
+    }
+  }
+
+  .dropdown-menu {
+    background: $room-card-background;
+    box-shadow: 0px 10px 20px opaque-black(0.25);
+    border-radius: $border-radius--lg;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    padding: $spacing--md $spacing--xl;
+
+    font-size: $font-size--lg;
+    font-weight: $font-weight--500;
+    line-height: 19px;
+
+    &:focus,
+    &:hover {
+      background: opaque-white(0.2);
+      box-shadow: 0px 2px 4px opaque-black(0.25);
+      border-radius: $border-radius--xl;
+    }
```

Comment:
Regullar vs Spaces dropdown style inconsistencies. Might be best if the spaces one follows the focus and hover styles of the regular dropdown even if the regular ones might need adjustments like a dotted instead of solid border or something else.


![sparkle-dropdown-02](https://user-images.githubusercontent.com/79229621/135423645-664a83ae-6b70-4ca6-857f-5454069976a0.png)
![sparkle-dropdown-01](https://user-images.githubusercontent.com/79229621/135423659-5e0f9acc-c0f2-4872-b056-4d3f7133cbb0.png)

![sparkle-dropdown-04](https://user-images.githubusercontent.com/79229621/135424213-4c5eba51-f362-4254-8d80-7941b34c542c.png)
![sparkle-dropdown-03](https://user-images.githubusercontent.com/79229621/135424223-dea57710-c005-47d6-8854-51290d8f3ff7.png)

Reply:
Fair comment. Looks like our dropdowns have different styles.

I've left @debt comments for all used dropdowns to align their styles later 51960bedd4f81f473bff3015b7d243122299cf41